tools/api-markdown-documenter/src/renderers/markdown-renderer/default-renderers/RenderHeading.ts

```diff
@@ -65,10 +64,23 @@ function renderHeadingWithMarkdownSyntax(
 		}
 	} else {
 		if (headingNode.id !== undefined) {
-			renderAnchor(headingNode.id, writer, context);
+			renderAnchor(headingNode.id, writer);
 		}
 		renderNodes(headingNode.children, writer, { ...context, bold: true });
 	}
 
 	writer.ensureSkippedLine(); // Headings require trailing blank line
 }
+
+/**
+ * Renders an HTML anchor for the given ID.
+ *
+ * @param anchorId - The ID of the associated item.
+ * @param writer - Writer context object into which the document contents will be written.
+ * @param context - See {@link RenderContext}.
+ */
+function renderAnchor(anchorId: string, writer: DocumentWriter): void {
+	writer.ensureNewLine(); // Ensure line break before tag
+	writer.write(`<a name="${anchorId}" />`);
+	writer.ensureNewLine(); // Ensure line break after tag
+}
```

Comment:
I was thinking that in an ideal world I'd expect to see anchors rendered in markdown with the usual syntax... which I then realized actually means "headings (and only headings) are anchors". But I guess anchors don't _have_ to be tied to headings, and this gives us that flexibility?

Reply:
Exactly. Both HTML and Markdown only support heading levels 1-6. Our system tries to offer more flexibility than that by rendering higher level "headings" as bold text with anchors.

Note that this is existing behavior - just moved the function to be more local now that it's only used in 1 place.